src/components/examples/footer.vue

```diff
@@ -25,13 +25,19 @@ export default {
 </style>
 
 <style lang="stylus">
++layout(xs)
+  .ex-attributes .zi-card
+    display flex
+    overflow scroll
+    &::after
+      content ''
+      padding-right var(--geist-gap)
```

Comment:
A blank line is required between two **style code blocks**.
```suggestion
      padding-right var(--geist-gap)
      
```